### DISCUSSION TITLE
Application Problem

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#832] [Q&A] [answered]
Hi, If I want to realization the autonomous fly with Loco position system and I followed the instruction setup the node before. So can I only use example of python do setup the set point to fly. Is it correct ? Or I must through the client software to realization ? thanks.

**Comment by knmcguire** [comment, marked as answer]
Within the client it is only possible to fly with a [controller in position mode](https://www.bitcraze.io/documentation/tutorials/getting-started-with-flying-using-lps/) or to use the [command based flight controller.](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/flightcontrol_tab/). 

For anything more complicated, you should use a python script for that. We have several [step-by-step guides](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/) and you can check out some [autonomy example scripts](https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples/autonomy) as well, which use several functions that have been documented in the [API reference](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/api/cflib/)

**Reply by CalvinKoung** [reply]
ok, I got it. Thanks your help.
